### DISCUSSION TITLE
Propose Ideas or Topics for Monthly RC-HTA-WG Meeting

### DISCUSSION THREAD

**Discussion by hoppanda** [RConsortium/HTA-wg#3] [General]
Please **`reply within`** each monthly RC-HTA-WG meeting to propose ideas and topics to the agenda

**Comment by hoppanda** [comment]
Call for ideas and topics for RC-HTA-WG meeting at Nov 27, 2024

**Reply by jSoboil** [reply]
Development of standardizing an R project for HTA, like what golem or devtools does for package development. In other words, automate the initiation and creation of a standardized file structure for analysts to use.

**Comment by hoppanda** [comment]
Call for ideas and topics for RC-HTA-WG meeting at Feb 20, 2025

**Reply by jrdnmdhl** [reply]
@hoppanda Might be worthwhile to talk about generative AI.

Some potential sub-issues:
- Should be we be planning for genAI playing a large role in HTA?
- How will genAI differentially affect work in R vs. work in legacy HTA software (e.g. Excel, SAS, ...)? Is it a threat or an opportunity?
- How do we build packages that genAI can use effectively and where work done by genAI can be efficiently checked?

**Reply by hoppanda** [reply]
will add! thx

**Comment by hoppanda** [comment]
Call for ideas and topics for RC-HTA-WG meeting at March 20, 2025
